lib/phoenix_html.ex

```diff
@@ -63,33 +63,33 @@ defmodule Phoenix.HTML do
   @typedoc "May be safe or unsafe (i.e. it needs to be converted)"
   @type unsafe  :: Phoenix.HTML.Safe.t
 
-  @doc """
+  @doc ~s'''
   Provides `~e` sigil with HTML safe EEx syntax inside source files.
 
   Raises on attempts to interpolate with `\#{}`, so `~E` should be preferred.
 
-      iex> ~e"\""
+      iex> ~e"""
       ...> Hello <%= "world" %>
-      ...> "\""
-      {:safe, [[["" | "Hello "] | "world"] | "\\n"]}
+      ...> """
+      {:safe, "Hello world\\n"}
```

Comment:
I'm not sure about this example now. It might be a good idea to show some more complex scenario that won't produce a single binary so that people don't assume it will always return a string.